libraries/wutsocket/wut_socket.c

```diff
@@ -0,0 +1,80 @@
+#include "wut_socket.h"
+
+static devoptab_t
+__wut_socket_devoptab =
+{
+   .name = "soc",
```

Comment:
Does this have to be 3 letters? could be `sock` or `socket` for extra Descriptiveness

Reply:
Yeah, that's a possibility.
Shouldn't have a very big impact though, since applications never reference the socket device path anyway

Reply:
Well I guess there's also no reason _not_ to do it so there you go!